rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/view.vm

```diff
@@ -83,7 +83,7 @@
 				
 				#set( $value= ["#formatDisplayName(${currentRWikiObject.user})","#formatDateTime($currentRWikiObject.version)"])
 				${rlb.getFormattedMessage("jsp_last_modified_by", $value.toArray())}
-				<a id="toCommentsLink" href="#tocomments" style="display:none" style="padding:0 1em 0 0">${rlb.jsp_comment_gotolink} <span></span></a>
+				<buttona id="toCommentsLink" href="#tocomments" style="display:none" style="padding:0 1em 0 0">${rlb.jsp_comment_gotolink} <span></span></a>
```

Comment:
buttona?

Reply:
lol. Well spotted. It's a new webcomponent called buttona